index.html

```diff
@@ -528,9 +528,13 @@
                 <p>Talent consultants and recruiters/matchmakers focused on supporting Michigan technology companies with their talent needs.</p>
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
+                <a href="https://angel.co/ann-arbor">AngleList - Ann Arbor Startups</a>
```

Comment:
Fixed this commit so instead we don't change the text, we instead change the link!